src/views/homepage/helpers/transform-raw-content.ts

```diff
@@ -11,12 +11,15 @@ async function transformRawContent(
 	authoredContent: HomePageAuthoredContent
 ): Promise<HomePageContentProps> {
 	// Destructure data needed from given `authoredContent`
-	const { hero, navNotice, merchandising, learnSection, preFooter } =
+	const { hero, navNotice, merchandising, certificationsSection, preFooter } =
```

Comment:
Transforming the raw content changes very slightly - really just variable name changing here.